ccmlib/scylla_node.py

```diff
@@ -714,11 +714,19 @@ def import_bin_files(self):
                 self._launch_env = dict(os.environ)
                 self._launch_env['LD_LIBRARY_PATH'] = dbuild_so_dir
 
-                cmd = [loader, os.path.join(dbuild_so_dir, 'patchelf'), '--set-interpreter', loader, dst]
-                p = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, env=self._launch_env)
-                (stdout, stderr) = p.communicate()
-                if p.returncode != 0:
-                    raise RuntimeError('{}: exited with status {}.\nstdout:{}\nstderr:\n{}'.format(cmd, p.returncode, stdout, stderr))
+                stripped = False
+                while True:
```

Comment:
It's nicer to extract the snippets into small local functions than to code this as a loop and force the reader to understand that it's "run at most twice".

Reply:
ok, fixed in v3